docs/src/main/sphinx/release/release-392.md

```diff
@@ -0,0 +1,105 @@
+# Release 392 (3 August 2022)
+
+## General
+
+* Add support for dynamic filtering with task retries. ({issue}`9935`)
+* Add support for using correlated sub-queries with ``DELETE`` queries. ({issue}`9447`) 
+* Fix potential query failure in certain complex queries with multiple joins and
+  aggregations. ({issue}`13315`)
+
+## JDBC driver
+
+* Add the ``assumeLiteralUnderscoreInMetadataCallsForNonConformingClients``
+  configuration property as a replacement for
+  ``assumeLiteralNamesInMetadataCallsForNonConformingClients``, which is
+  deprecated and planned to be removed in a future release. ({issue}`12761`)
+
+## ClickHouse connector
+
+* Add report for total time elapsed reading data from JDBC sources. ({issue}`13132`)
+
+## Delta Lake connector
+
+* Add support for using a randomized location when creating a table, so that
+  future table renames or drops do not interfere with new tables created with
+  the same name. This can be disabled by setting the
+  ``delta.unique-table-location`` configuration property to false. ({issue}`12980`)
+* Add ``delta.metadata.live-files.cache-ttl`` configuration property for the
+  caching duration of active data files. ({issue}`13316`)
+* Retain metadata properties and column metadata after schema changes. ({issue}`13368`, {issue}`13418`)
+* Prevent writing to a table with `NOT NULL` or
+  [column invariants](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-invariants)
+  columns. ({issue}`13353`)
+* Fix incorrect min and max table statistic values when writing `NULL`. ({issue}`13389`)
+
+## Druid connector
+
+* Add support for timestamp predicate pushdown. ({issue}`8404`)
```

Comment:
```Support predicate pushdown on `timestamp` columns```

`timestamp` is a specific datatype, we don't support for all types which might represent timestamps (like `TIMESTAMP WITH TIME ZONE` etc.).

Reply:
The type is actually `timestamp(p)` or `timestamp(p) without time zone`